contributor-growth/drafts/recruiting-playbook.md

```diff
@@ -0,0 +1,133 @@
+# Recruiting Contributors
+*Status: Draft; Missing Components*
+/help-wanted
+
+## Summary 
+
+Recruiting and retention go hand in hand. If you are thinking "how can we add 
+more amazing contributors?" and "where do we find them?" the projects leadership
+ needs to think about how you keep folks around, too. If your goal is to limit 
+ to only single contributions aka drive-bys then this might not apply to you. 
```

Comment:
I don't understand what this line means about "limit to only single contributions".